### ISSUE TITLE
question about the loss function

### ISSUE THREAD

**Issue by OValery16** [junyanz/BicycleGAN#33]
Dear authors,
thanks for your amazing work. 

I have a question about the value of the lambdas in the loss function. I understood that they define a certain balance between converging to several images close to the original B, and diverging a bit from the original B.

By default, you chose:

lambda_L1=10.0
lambda_GAN=1.0
lambda_GAN2=1.0
lambda_z=0.5
lambda_kl=0.01

I trained the bicycle gan model with the default parameters. My dataset consist of 90 000 images. Meanwhile some result are impressive, the majority remain a bit blurry (they don t follow the same probabilty distribution as in my original dataset). I guess the lambda_L1 is not high enough to force the network to generate images that follow the original probability distribution. What result could you get if you choose a lambda_L1=100 such as in pix2pix ?

**Comment by junyanz** [comment]
We use Lambda_l1=10 as we are using the least square objective, which has a smaller range compared to the cross-entropy objective. In general, a higher lambda_L1 loss will produce more blurry results. You can see the original [pix2pix](https://arxiv.org/abs/1611.07004) paper for a comparison with and without GAN loss.

**Comment by OValery16** [comment]
Thanks for your answer.
